### ISSUE TITLE
Error in downloading Ncv6gt2Luew: request() got an unexpected keyword argument 'stream'; continuing to next video. ...

### ISSUE THREAD

**Issue by jiangjianshan** [learningequality/ka-lite#5188]
I have just updated KA Lite from 0.16.5 to 0.16.6 and has run the command "kalite manage setup" on the command prompt windows. It has been installed in Windows 7 64 bit environment. The Python version is 2.7.10.  I think it should meet the requirement.txt file as it written.
    But when I want to download the video,  I have got the following issue:
Error in downloading Ncv6gt2Luew: request() got an unexpected keyword argument 'stream'; continuing to next video. ...

![image](https://cloud.githubusercontent.com/assets/20179047/16404632/5b2091ea-3d33-11e6-8c34-ebf32df08f73.png)

![image](https://cloud.githubusercontent.com/assets/20179047/16404638/6b762e60-3d33-11e6-8936-47d4d83639ae.png)

**Comment by cpauya** [comment]
Thanks fro reporting @jiangjianshan - we will try to reproduce your issue to come-up with a fix.

I'm confused with your environment: You mentioned you were using Windows 7 64-bit but the Console window above shows Windows Server 2008 64-bit, can you verify which one?

Also, can you specify which browser / version did you use for this?

Thanks again for the report.

**Comment by arceduardvincent** [comment]
Hi @cpauya and @jiangjianshan.
Following the steps above ` 0.16.5` to `0.16.6`
I replicate similar issue on my virtualbox. `Windows Ultmate` 64-bit. 
![screen shot 2016-06-28 at 5 11 05 pm](https://cloud.githubusercontent.com/assets/4099119/16410096/643519e8-3d53-11e6-9a33-b8068b1a975f.png)

**Comment by djallado** [comment]
@arceduardvincent: The same issue when I use `IE11 Windows 7 32-bit`.
Here's the snapshot:
![screen shot 32-bit](https://cloud.githubusercontent.com/assets/8664071/16414347/bae0170a-3d69-11e6-80c7-c560fda3297f.png)

**Comment by radinamatic** [comment]
I'm also unable to replicate this with 0.16.6 on Win 7 Ultimate 64 bits, IE11 - videos download correctly, no errors reported:

![win7666 0 16 6post1 running - oracle vm virtualbox_507](https://cloud.githubusercontent.com/assets/1457929/16416651/9360d5e6-3d43-11e6-91a2-12cd91ebb27b.png)

**Comment by benjaoming** [comment]
This is because the wrong version of `requests` is in use.

It could be an artifact because the previous version bundled `requests` in `python-packages`, you need to possibly remove *pyc files during the install process if it's overwriting existing locations.

**Comment by benjaoming** [comment]
I think my above comment is ascertained by @radinamatic 's inability to reproduce with a fresh version.

**Comment by benjaoming** [comment]
Another reason for #3952

**Comment by jiangjianshan** [comment]
Hello, @cpauya, my laptop was running on Windows 7 Professional 64 bit.  The Console window above shows Windows Server 2008 64-bit is the Windows SDK 7.0 console window. I just use it to show the version of python package I'm using.
Also, I use the Google chrome which version is 51.0.2704.103 m (64-bit). It seems like the problem is because of the version of requests package, but I have update it to 2.10.0 and not work.
I have also done the following steps:
1. Removed the whole folder of .kalite and only remaining the .kalite\content folder which containing the videos I have download by Kalite which version is 0.16.5.
2. To use the command "pip uninstall ka-lite-static" to uninstall the ka-lite package on the Python 2.7.10.
3. Double click the KALiteSetup-0.16.6.exe to try to make a fresh installation.
4. After finished the installation, I run the command "kalite manage setup".
    Finally, I still got the same issue I have report at here. Do I need to uninstall the whole python installed program and reinstall python 2.7.10 again? And then to do the step 3 to step 4?

**Comment by djallado** [comment]
@jiangjianshan: Thank for the additional information. @benjaoming gave suggestion above, I'll check this for now.

**Comment by jiangjianshan** [comment]
Hello all, finally I have solved the issue happen before. The steps what I have done are as below:
1. Uninstall KALite 0.16.6 program.
2. Uninstall Python 2.7.10.
3. Changed the .kalite folder to another name to backup the videos.
4. Reboot the laptop, and then delete installation folder of Python 2.7.10, which contain the files haven't been deleted by the uninstaller.
5. Install the KALiteSetup-0.16.6.exe to do a fresh version installation.
6. Copied the backup videos into the new .kalite/content folder.
6. Run the command "kalite manage setup".
    After a few hours, I start the kalite, now it is work correctly. The videos has been started to download.
